src/models/sunburst.js

```diff
@@ -60,16 +102,6 @@ nv.models.sunburst = function() {
 
             var g = wrapEnter.selectAll('nv-sunburst');
 
-            chart.update = function() { 
-                if ( duration === 0 ) {
-                    container.call(chart);
-                } else {
-                    container.transition().duration(duration).call(chart);
-                }
-            };
-            chart.container = this;
-
-
```

Comment:
I removed this, because it didn't solve the problem. In fact it wasn't called ...